Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -7,6 +7,7 @@ import UIKit
 private enum Layout {
   enum Style {
     static let cornerRadius: CGFloat = Styles.grid(2)
+    static let modalHeightMultiplier: CGFloat = 0.65
```

Comment:
This really only applies to users with the default sized font. If you have larger fonts in use the view controller height will expand up to a certain point.